tests/runner.py

```diff
@@ -24,7 +24,9 @@
 
 
 def find_all_tests(excludes=[]):
-    tests = list(map(lambda p: p.strip('/').split('/')[-1], glob.glob('tests/*_[0-9]*')))
```

Comment:
You can do like this:

`tests = list(map(lambda p: p.strip('/').split('/')[-1], sorted(glob.glob('tests/*_[0-9]*'))))`

Reply:
Adding `sorted` to make it sort and one-line :)

Reply:
Brilliant! That's what truly prettification. I've done that.